.gitignore

```diff
@@ -8,4 +8,4 @@ config/
 .envrc
 # Default path for Juno DB files.  It will get created if you follow the
 # README and/or run `./build/juno` command.
-juno/
+.juno/
```

Comment:
why do we need this change?

Reply:
https://github.com/NethermindEth/juno/pull/2011/commits/4aad791c01e513476c80363567054f06f7c3e3b7

You can see the commit message, basically without this the files in `cmd/juno` would be ignored

Reply:
it shouldn't be the case. For instance here is a PR that change files in the directory - https://github.com/NethermindEth/juno/pull/1902/files

Btw If I specify `.dir` in .gitignore does it mean that it will ignore hidden directory or `./dir/` basically?

Reply:
ha, maybe I missed something here, let me revert this